[FEATURE] New colorscheme "totalblack"

3mpee3mpee opened the pull request:
new colorscheme for the deep dark colors inspired by [Vanta Black HyDe theme](https://github.com/rishav12s/Vanta-Black) and [Onedark Dark](https://github.com/olimorris/onedarkpro.nvim)

<img width="2536" height="1381" alt="image" src="https://github.com/user-attachments/assets/e6db4f47-953e-4d5c-b98e-84632cf354e8" />
<img width="2533" height="1381" alt="image" src="https://github.com/user-attachments/assets/aaa3cc21-4a58-4e9c-93a1-19ee01de799b" />
<img width="2537" height="1379" alt="image" src="https://github.com/user-attachments/assets/a6be1478-82fc-4271-9d26-19861bc898fa" />

3mpee3mpee committed:
new colorscheme for the deep dark colors

Jelmerro commented on app/colors/totalblack.css:
```diff
@@ -0,0 +1,150 @@
+* along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+/*
+ * COLORS
```
Since there are only colors, this comment is redundant.

Jelmerro reviewed (commented):
Thanks for taking the time to add this new theme. I think should could be a nice addition, specifically for OLED screens and the like, but there are some small things I would like to have changed. Overall, I think the theme looks really good.

3mpee3mpee committed:
colorscheme "totalblack" updates

1) Changed Copyright year from "2019-2025" -> "2025"
2) Styles duplicates clean up

Jelmerro reviewed (commented):
Thanks for the update, this looks really great and compact now. Just one more thing I'm a bit concerned about, is that there is no separate color for suspended tabs. These are by default `#000`, but with this theme any tab is, so a different color for suspended ones would be appreciated, though I'm not sure which one would look nice. (oh, and just to make sure, if you now update the file, it changes from just `2025` to `2025-2026` at the top 😉)

3mpee3mpee committed:
Merge branch 'master' into feature/totalblack

3mpee3mpee commented:
> Thanks for the update, this looks really great and compact now. Just one more thing I'm a bit concerned about, is that there is no separate color for suspended tabs. These are by default `#000`, but with this theme any tab is, so a different color for suspended ones would be appreciated, though I'm not sure which one would look nice. (oh, and just to make sure, if you now update the file, it changes from just `2025` to `2025-2026` at the top 😉)

You right, I checked couple of different colors, for me #111 works well, so updated with it. Let me know if you have any  concerns. And Happy New Year btw !!!

3mpee3mpee committed:
colorscheme totalblack: updated year & tab-suspended value
